configs/mainnet/altair.yaml

```diff
@@ -22,8 +22,8 @@ INACTIVITY_SCORE_BIAS: 4
 
 # Time parameters
 # ---------------------------------------------------------------
-# 2**8 (= 256)
-EPOCHS_PER_SYNC_COMMITTEE_PERIOD: 256
+# 2**9 (= 512)
+EPOCHS_PER_SYNC_COMMITTEE_PERIOD: 512
```

Comment:
We need to force `LIGHT_CLIENT_UPDATE_TIMEOUT` to be equal to the sync committee period.

Tbh, might jsut remove `LIGHT_CLIENT_UPDATE_TIMEOUT` and just use `EPOCHS_PER_SYNC_COMMITTEE_PERIOD` in it's place